scss/titles.scss

```diff
@@ -13,3 +13,16 @@
     z-index: -1;
   }
 }
+
+@media screen and (max-width: 1024px){
```

Comment:
why do you need this media query?. `Front-End and Back-End` titles should not move when the screen width changes.